packages/gatsby-plugin-sharp/src/image-data.ts

```diff
@@ -7,6 +7,7 @@ import { traceSVG, getImageSizeAsync, base64, batchQueueImageResizing } from "."
 import type { SharpInstance } from "sharp"
 import sharp from "./safe-sharp"
 import { createTransformObject } from "./plugin-options"
+import { meta } from "joi"
```

Comment:
Not sure how this got in here. Will remove next week.

Reply:
Accidental import while typing "metadata" is my guess